server/routes/getting_started/getting_started_router.ts

```diff
@@ -134,4 +129,75 @@ export function registerGettingStartedRoutes(router: IRouter) {
       }
     }
   );
+
+  router.post(
+    {
+      path: `/api/observability/gettingStarted/createAssets`,
+      validate: {
+        body: schema.object({
+          mdsId: schema.string(),
+          mdsLabel: schema.string(),
+          tutorialId: schema.string(),
+        }),
+      },
+    },
+    async (
+      context,
+      request,
+      response
+    ): Promise<IOpenSearchDashboardsResponse<any | ResponseError>> => {
+      try {
+        const { mdsId, mdsLabel, tutorialId } = request.body;
+        const fileData = await loadAssetsFromFile(tutorialId);
+
+        const objects = await createSavedObjectsStreamFromNdJson(Readable.from(fileData));
+        const loadedObjects = await objects.toArray();
+
+        const updatedObjects = loadedObjects.map((obj) => {
+          if (mdsId) {
+            const newId = `mds-${mdsId}-objectId-${obj.id}`;
+
+            const newReferences =
+              obj.references?.map((ref: { type: string; id: any }) => {
+                if (ref.type === 'visualization') {
```

Comment:
Do we need to change this for index patterns too? When it comes to creation to visualizations from the ndjson assets?